packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -713,6 +717,7 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
         debugPrintStack(label: 'scheduleFrame() called. Current phase is $schedulerPhase.');
       return true;
     }());
+    ensureFrameCallbacksRegistered();
```

Comment:
It feels a little bit hacky to me that we prevent the engine to achieve frame by not attaching listener, but i can't think of a better way, either.